frontend/src/screens/WelcomeScreen.js

```diff
@@ -1,63 +1,50 @@
-import React, { Component } from "react";
+import React, { useState, useContext } from "react";
 import Button from "@material-ui/core/Button";
 import Typography from "@material-ui/core/Typography";
 import "./WelcomeScreen.css";
 import { MyContext } from "../infra/MyContext";
 import TextField from "@material-ui/core/TextField";
 
-class WelcomeScreen extends Component {
-  constructor() {
-    super();
-    this.state = {
-      roomToJoin: ""
-    };
-  }
-  onChange = event => {
+const WelcomeScreen = () => {
+  const context = useContext(MyContext);
+  console.log(context.state.phase);
+  const [roomToJoin, setRoom] = useState("");
+
+  const onChange = (event) => {
     const value = event.target.value;
-    console.log(this.state.roomToJoin);
-    this.setState({ roomToJoin: value });
+    setRoom(value);
```

Comment:
what does that mean?

Reply:
it more easy to debug that why but i eraesed it